README.md

```diff
@@ -209,12 +209,11 @@ would involve:
   class Scribe_module_1(ScribeModuleBaseClass):
 
       def __init__(self, input_dict=None, module_name=None, host_name=None,
-                   input_type=None, scribe_uuid=None):
```

Comment:
What if someone isn't using our work?

Reply:
ES has a `_id`
```
  "_id": "4538dbdefd127dabecb3a6fde17f828978a5245cc5e9e5525362ddbc6059d37d",
  "_version": 1,
  "_score": null,
  "_source": {
    "module": "cpuinfo",
    "host": "localhost",
    "source_type": "stockpile",
    "scribe_uuid": "9c368a0c-0156-473b-9e29-8fcd801046ca",
    "value": {
      "Architecture": "x86_64",
      "CPU op-mode(s)": "32-bit, 64-bit",
      "Byte Order": "Little Endian",
      "CPU(s)": "8",
```

Reply:
yeah, that _id is now generated by stockpile-wrapper https://github.com/cloud-bulldozer/bohica/blob/master/stockpile-wrapper/stockpile-wrapper.py#L65

Reply:
ack!